pkg/graph/schema.graphql

```diff
@@ -1457,6 +1462,8 @@ type Query {
   @hasAnyRole(roles: [MINT_USER, MINT_MAC])
   existingModelLink(id: UUID!): ExistingModelLink!
   @hasAnyRole(roles:[MINT_USER, MINT_MAC])
+  searchChanges(filters: [SearchFilter!], limit: Int!, offset: Int!): [ChangeTableRecord!]!
+  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
   searchChangeTable(request: SearchRequest!, limit: Int!, offset: Int!): [ChangeTableRecord!]!

```

Comment:
I think it would make sense to remove the other queries at this point.

Reply:
I missed that you did do this in the resolver, so I think that this is could be ok as is. It's probably good to deprecate it or remove them sometime though.